tests/data/format/summary_splitter/dots.py.out

```diff
@@ -0,0 +1,16 @@
+class MyClass:
+    """We shouldn't split sys.path."""
+
+
+def func():
+    """We should not add extra lines after the dot
+    to it.
+    """
+
```

Comment:
```suggestion
def func():
    """We should not add line after i.e., sys.path, e.g., etc. but etc. in particular 
    
    is harder right ?
    """
```
Just wondering what happens here 😄 I'm sure it's not a very important use case.

Reply:
If my regex skill haven't failed me this should pass 😄 

I'll check!